wiki/Article_styling_criteria/Writing/en.md

```diff
@@ -336,9 +336,9 @@ Contractions must not be used.
 
 The serial comma, also known as the Oxford or Harvard comma, must be used.
 
-### Em dash
+### Dashes
 
-Em dashes (`—`) must be used without spaces around them. Hyphens must not be used in their place.
+Refer to [Wikipedia's Manual of Style for dash usage](https://en.wikipedia.org/wiki/Wikipedia:Manual_of_Style#Dashes). When punctuating a sentence however, use em dashes with spaces around them (" — "), not spaced en dashes (" – "). Hyphens ("-") must not be used in place of dashes.
```

Comment:
> Hyphens must not be used in place of dashes.

I hope "must" here means that they will be *eventually* converted to dashes, not that it *must* be done at the review stage, since in #7376 most reviewers agree it's not a tragedy if new hyphens pop up

Reply:
there's no consequence for not following the asc. if no one catches it, the same could be said for any "rule" here

Reply:
right

Reply:
in practice "must" in asc is like "if the reviewer(s) feel it's important enough to correct at this time" so I don't think there is much of a difference here